libs/util/assertUtils.hpp

```diff
@@ -60,10 +60,11 @@ inline void printCallStack() {
           free(ret);
         }
       }
-      LOG_FATAL(GL, "\n" << os.str());
+      LOG_INFO(GL, "\n" << os.str());
```

Comment:
Why did you change from FATAL to INFO?

Reply:
Because I added call stack prints while debugging and didn't want my tests to fail.

Reply:
reverted